### PR TITLE
Register the user for minting regularly

### PR DESCRIPTION
**Pull request by shelbyd**
This registers whenever we get new data as long as we're not currently registered.

### CODE REVIEW

File: src/services/ProtocolService/index.ts [hunk 1]
```diff
@@ -41,29 +45,61 @@ export default class ProtocolService {
...
-              const { name, section } = decoded;
-              const error = `ProtocolService.registerForMinting error: ${section}.${name}`;
-
-              console.error(error);
-
-              reject(error);
-            }
+      const txn = this.api.tx.fractalMinting.registerForMinting(null, proof);
+      txn.signAndSend(this.signer, (result) => {
+        if (result.status.isFinalized) return resolve();
```
Comment (frm) [inline src/services/ProtocolService/index.ts]: Can we get the hash from `txn`? If so, maybe we should give it as an argument to `resolve`

File: src/services/ProtocolService/index.ts [hunk 2]
```diff
@@ -41,29 +45,61 @@ export default class ProtocolService {
...
+            reject(error);
           }
-        });
+        }
+      });
     });
   }
 
+  public async isRegisteredForMinting(accountId: string): Promise<boolean> {
+    return false;
```
Comment (frm) [inline src/services/ProtocolService/index.ts]: I'm assuming this is temporary? Can we add a `TODO` here?

**Review by frm — CHANGES_REQUESTED** [review]

File: src/services/ProtocolService/index.ts (hunk 2 above)
Comment (shelbyd) [reply src/services/ProtocolService/index.ts]: It was temporary. Removed now.

File: src/services/ProtocolService/index.ts (hunk 1 above)
Comment (shelbyd) [reply src/services/ProtocolService/index.ts]: Done.

**Pull request merged**